sdk/webpubsub/ci.yml

```diff
@@ -28,8 +28,11 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: webpubsub
+    TestProxy: true
     Artifacts:
     - name: azure-mgmt-webpubsub
       safeName: azuremgmtwebpubsub
     - name: azure-messaging-webpubsubservice
-      safeName: azuremessagingwebpubsubservice
\ No newline at end of file
+      safeName: azuremessagingwebpubsubservice
+    - name: azure-webpubsub-client
```

Comment:
Is there a reason this isn't called `azure-messaging-webpubsubclient`? Or was there a decision to drop the `messaging` namespace going forward?

Reply:
I remember @johanste agreed with the name: https://github.com/Azure/azure-sdk/issues/5623 although I can't find the specific words now...

Reply:
If that's what Johan agreed to then that's fine. Not sure when you plan to release the first beta package - he gets back from vacation next week so we can confirm then.

Reply:
If the other webpubsub package(s) use the messaging group name, we should do that here as well...

Reply:
Got it. Let us keep consistent with other language to add `messaging` in package name